packages/datadog-plugin-http2/src/server.js

```diff
@@ -16,7 +16,13 @@ class Http2ServerPlugin extends ServerPlugin {
     this.addSub(`apm:${this.constructor.id}:server:${this.operation}:${eventName}`, handler)
   }
 
-  start ({ req, res }) {
+  addTraceBind (eventName, handler) {
+    this.addBind(`apm:${this.constructor.id}:server:${this.operation}:${eventName}`, handler)
```

Comment:
Why is this needed in addition to the existing methods?

Reply:
`bindStart` and `bindFinish` never get called without adding this method.

Reply:
Yup, you're right, the naming was wrong, the `tracingPlugin` was using the channel prefix `apm:http2:request`, but the prefix should be `apm:http2:server:request`. I fixed it for http2 server and http server. And removed the addTraceSub method calls in the http plugins.